packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -69,6 +69,14 @@ yarn workspace v1.22.19image` is a collocated image path, this entry will be the
     [key: string]: unknown;
   };
 
+  export type FileChange = {
```

Comment:
I'm not fan of the historical name of this type

I think the original motivation was to support some kind of `history: FileChange[]`, but if you think about it, its purpose is only to be used in front matter because it has a broad type `date: Date | string` (suitable for Yaml automatic parsing of dates)

If we ever support a history metadata attribute, it is unlikely we'll use this type anyway, so I'd prefer to rename it as `FrontMatterLastUpdate` (for both docs and blog plugin), as it makes things more explicit this type is not meant to be used in other contexts